src/vmspawn/vmspawn.c

```diff
@@ -127,6 +131,7 @@ static int parse_argv(int argc, char *argv[]) {
                 { "qemu-kvm",        required_argument, NULL, ARG_QEMU_KVM        },
                 { "qemu-vsock",      required_argument, NULL, ARG_QEMU_VSOCK      },
                 { "vsock-cid",       required_argument, NULL, ARG_VSOCK_CID       },
+                { "vtpm",            required_argument, NULL, ARG_VTPM            },
```

Comment:
i'd just call this "tpm" btw. the fact that it is virtual is kinda implied i'd say

Reply:
Fixed in eae47ee9d9.